src/RemediationRoleSaver.cpp

```diff
@@ -102,7 +103,10 @@ void RemediationSaverBase::removeFileWhenEmpty(const QString& filename)
 
 void RemediationSaverBase::saveFileOK(const QString& filename)
 {
-    // TODO: if OK - inform the user
+    QMessageBox::information(
```

Comment:
`filename` was previously unused, so I added a information box. It helps with some of the "silent success" saves.